android/app/src/main/java/poke/rogue/helper/presentation/type/TypeSelectionStates.kt

```diff
@@ -29,11 +36,11 @@ data class TypeSelectionStates(
     val areAllTypesEmpty: Boolean
```

Comment:
```suggestion
    val isAllEmpty: Boolean
        get() = !isMyTypeSelected && !isOpponent1Selected && !isOpponent2Selected
```

이 정도로도 충분히 이해하기 쉬울 것 같습니다!